### PR TITLE
ExLlamav2_HF: Convert logits to FP32

### PR DESCRIPTION
**Pull request by turboderp**
ExLlama converts logits to FP32 at the end of the forward pass, while ExLlamaV2 returns FP16 logits straight from the lm_head layer. Converting in the V2 loader makes sure the HF sampler behavior the same for V1 and V2.

## Checklist:

- [X] I have read the [Contributing guidelines](https://github.com/oobabooga/text-generation-webui/wiki/Contributing-guidelines).

### CODE REVIEW

**Comment by Ph0rk0z** [comment]
PPL on evaluation went down slightly after this.

**Comment by oobabooga** [comment]
Thanks @turboderp, I would never have noticed this.
